mscore/musescore.cpp

```diff
@@ -1327,6 +1328,16 @@ MuseScore::MuseScore()
             cornerLabel->setPixmap(QPixmap(":/data/mscore.png"));
             cornerLabel->setGeometry(width() - 48, 0, 48, 48);
             }
+      QSettings s;
+      QString keyboardLayout;
+      if (!s.contains("keyboardLayout")) {
```

Comment:
Need a protection here for when called in command line mode, `if (!MScore::noGui)`, best around the entire block you added, in commandline mode shortcuts are not used and keyboardlayout isn't relevant, so it also should not try to save it in prefs, even on 1st time.